### DISCUSSION TITLE
Versioning issues

### DISCUSSION THREAD

**Discussion by rouke-broersma** [stryker-mutator/mutation-testing-elements#3045] [General]
v3 of mutation testing elements was recently released which included breaking changes in the implementations. However the schema has not been changed. We validate the version in the schema, so technically v3 implementations are not compatible with the current latest schema versions. This breaks the versioning schema we agreed on. See: https://github.com/stryker-mutator/mutation-testing-elements/blob/327d72de276d9d88c22846d82af509bd1597a656/packages/report-schema/src/mutation-testing-report-schema.json#L15

I think we need to revisit how we version mutation-testing-elements.

@hugo-vrijswijk @nicojs @xandervedder

**Comment by hugo-vrijswijk** [comment]
We should make a schema version 3 and unlock versioning (so schema will live on with its own versioning), and use semver 2.0.0
